### PR TITLE
improve upgrade to OS with tf, add missing resource imports

### PR DESCRIPTION
**Pull request by byashimov**
# What changed, and why it matters

Adds missing required imports to use OS with TF

### CODE REVIEW

File: docs/products/opensearch/howto/upgrade-to-opensearch-with-terraform.rst [hunk 1]
```diff
@@ -78,6 +78,14 @@ Bring the Terraform state back in sync with your OpenSearch service by importing
 
     terraform import 'aiven_opensearch.os' <project-name>/<service-name>
 
+Describe and import dependent resources if exist.
```
Comment (ivan-savciuc) [inline docs/products/opensearch/howto/upgrade-to-opensearch-with-terraform.rst:81]: I would move this to the end of the block and use the following working: 
If you have had any Elasticsearch ALCs and users, do not forget to import Opensearch counterparts to the Terraform state.

**Review by ivan-savciuc — CHANGES_REQUESTED** [review]
I think we should move it closer towards the end of the article after:

> Your Elasticsearch service has been upgraded to OpenSearch with Terraform, and the resource configuration updated to use a resource type of OpenSearch.

**Comment by ivan-savciuc** [comment]
@dewan-ahmed can you also check this one?

**Comment by dewan-ahmed** [comment]
Hi @ivan-savciuc @byashimov I'll leave the testing/validation of the steps to you. 

My suggestion would be to move this content from "OpenSearch/HowTo" to "Terraform/HowTo". The way most of our content are structured, they show how to do things using the console and then the tool-specific content are under docs/tools/<tool-name>, for example, docs/tools/terraform. 

https://docs.aiven.io/docs/products/opensearch/howto/upgrade-to-opensearch.html could have a link to this article. Check out how https://docs.aiven.io/docs/platform/howto/create_new_service.html has a note at the bottom of the article.

A minor editorial comment:

- "our Terraform provider" could be rephrased as "Aiven Provider for Terraform", the official name.

Thanks for this contribution.

**Review by dewan-ahmed — APPROVED** [review]
Thanks for these changes @byashimov 

LGTM.

**Pull request merged**